src/main/resources/data/reviews.csv

```diff
@@ -0,0 +1,101 @@
+Rating,Comment,Size,Length
```

Comment:
Are 100 reviews enough for the use cases you've got in mind? Do you think we need any more?

Reply:
The use case of the tutorial is leveraging the global rating and the size and length rankings. It is not leveraging the comments. So it is not an issue (at least for the use case of the tutorial) if the same comments appear several times.

Reply:
that makes sense, thanks - and we can always add to it in future if we need to illustrate use cases like sentiment analysis that do use the comment text